climakitae/data_export.py

```diff
@@ -167,28 +146,29 @@ def _export_to_user(user_export_format, data_to_export,
     ndims = len(data_to_export.dims)
```

Comment:
We might need to have some logic up front here to catch passing non-Xarray data to export. This `dims` call will only work with a Xarray `DataSet` or `DataArray` so the code will bomb here if passed something else. Also the logic later (line 186) won't have a chance to be called...

Reply:
thanks for pointing this out! I did indeed have the xarray object check too late

Reply:
thanks, I absolutely had that logic check in the wrong place, lol